libs/language-server/watcher/src/lib/watcher.ts

```diff
@@ -6,16 +6,37 @@ import { getNxVersion } from '@nx-console/language-server/workspace';
 import { gte } from 'semver';
 import type { WatchEvent } from 'nx/src/native';
 import { debounce } from '@nx-console/shared/utils';
+import { minimatch } from 'minimatch';
+
+const NX_PLUGIN_PATTERNS_TO_WATCH = [
+  '**/cypress.config.{js,ts,mjs,cjs}',
+  '**/{detox.config,.detoxrc}.{json,js}',
+  '**/app.{json,config.js}',
+  '**/jest.config.{cjs,mjs,js,cts,mts,ts}',
+  '**/next.config.{js,cjs,mjs}',
+  '**/nuxt.config.{js,ts,mjs,mts,cjs,cts}',
+  '**/playwright.config.{js,ts,cjs,cts,mjs,mts}',
+  '**/remix.config.{js,cjs,mjs}',
+  '**/.storybook/main.{js,ts,mjs,mts,cjs,cts}',
+  '**/{vite,vitest}.config.{js,ts,mjs,mts,cjs,cts}',
+  '**/webpack.config.{js,ts,mjs,cjs}',
+  '**/jest.preset.js',
+  '**/tsconfig.*.json',
+  // nx-dotnet
+  '*{.csproj,fsproj,vbproj}',
+];
```

Comment:
We should get this from the daemon eventually. We should be able to get the plugin globs there. We'll do this later